Core/Core/Extensions/URLExtensions.swift

```diff
@@ -55,8 +55,9 @@ extension URL {
     }
 
     public func appendingQueryItems(_ items: URLQueryItem...) -> URL {
+        let names = items.map { $0.name }
         var components = URLComponents.parse(self)
-        components.queryItems = (components.queryItems ?? []) + items
+        components.queryItems = (components.queryItems ?? []).filter { !names.contains($0.name) } + items
```

Comment:
Is this the expected behavior for array parameters too? Would we ever expect to append an `include[]`?

Reply:
that's a good question I hadn't considered.. seems dangerous now. maybe I shouldn't force the replace here and instead only do it in `appendingOrigin`.

Reply:
Actually, I think I know a better way. I'll just update the `originIsModuleItemDetails` check.